### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by jhabdas** [roylee0704/react-flexbox-grid#70]
I recently installed this dependency, which looks to be the leader in React Flexbox Grid systems, to my project to spike on a prototype and, after resolving #28, encountered the following:

```
Error in ./~/react-flexbox-grid/lib/components/Grid.js
Module not found: Error: Cannot resolve module 'style' in /Users/jhabdas/Developer/project/node_modules/react-flexbox-grid/lib/components
 @ ./~/react-flexbox-grid/lib/components/Grid.js 18:19-41

Error in ./~/react-flexbox-grid/lib/components/Col.js
Module not found: Error: Cannot resolve module 'style' in /Users/jhabdas/Developer/project/node_modules/react-flexbox-grid/lib/components
 @ ./~/react-flexbox-grid/lib/components/Col.js 14:19-41

Error in ./~/react-flexbox-grid/lib/components/Row.js
Module not found: Error: Cannot resolve module 'style' in /Users/jhabdas/Developer/project/node_modules/react-flexbox-grid/lib/components
 @ ./~/react-flexbox-grid/lib/components/Row.js 18:19-41
```

My environment:

```
node -v && npm -v
v6.3.1
3.10.3
```

It seems there are some installation issues with this lib and its dependencies. As a temporary workaround I was able to get past the issues by navigating into `node_modules/react-flexbox-grid` and issuing an `npm i` from that directory.

**Comment by wezlar** [comment]
I had the same issue as @jhabdas, the `npm i` seemed to do the trick in getting it working. Would be nice if this could be fixed.

**Comment by njwest** [comment]
Ran into same issue today, thanks for the fix @jhabdas

**Comment by chilijung** [comment]
I've found out the problem, you forget to install `style-loader` in your project! And you should install both `style-loader` and `css-loader`.

**Comment by jhabdas** [comment]
@chilijung Nice spot! I thought about submitting a pull to add the loaders to the `peerDependencies` section of the package manifest, but it'd be best not to place a hard requirement on Webpack. As a result, I simply updated the documentation. This issue will be closed automatically when #73 or https://github.com/jhabdas/react-flexbox-grid/commit/57409a7ece24599e9afa2185ce6ed0df30786beb is merged.
